wagtail_automatic_redirects/signal_handlers.py

```diff
@@ -75,9 +76,7 @@ def create_redirect_object_after_page_move(sender, **kwargs):
 # Register receivers
 def register_signal_handlers():
     page_published.connect(create_redirect_object_if_slug_changed)
-
```

Comment:
I adjusted the imports and logic a bit because we are limiting the Wagtail version to 2.15-3.0